source/_integrations/influxdb.markdown

```diff
@@ -35,43 +37,60 @@ influxdb:
 You will still need to create a database named `home_assistant` via InfluxDB's command line interface. For instructions on how to create a database check the [InfluxDB documentation](https://docs.influxdata.com/influxdb/latest/introduction/getting_started/#creating-a-database) relevant to the version you have installed.
 
 {% configuration %}
+api_version:
+  type: string
+  description: API version to use.  Valid values are `1` or `2`
+  default: "1"
+ssl:
+  type: boolean
+  description: Use HTTPS instead of HTTP to connect. **2.xx - Defaults to `true` for 2.xx, not `false`.**
+  required: false
+  default: false
 host:
   type: string
-  description: IP address of your database host, e.g., 192.168.1.10
+  description: IP address or domain of your database host, e.g., 192.168.1.10. **2.xx - Defaults to 'us-west-2-1.aws.cloud2.influxdata.com' for 2.xx, not 'localhost'.**
```

Comment:
This does not need emphasis, just write it in the normal text would do.

The same applies to all other occasions as well.

Reply:
Ok will remove the emphasis.